src/pushsource/_impl/backend/pub_source/pub_client.py

```diff
@@ -0,0 +1,66 @@
+import json
+import logging
+import os
+import threading
+import six
+
+import requests
+from more_executors import Executors
+
+if six.PY2:  # pragma: no cover
```

Comment:
I guess this will conflict now as the "remove py2 support" change was submitted, which removes six.